src/app.rs

```diff
@@ -287,8 +255,8 @@ impl<Ms, Mdl, INodes: IntoNodes<Ms> + 'static, GMs: 'static> App<Ms, Mdl, INodes
                     let mut new_effects = self.process_queue_message(msg);
                     queue.append(&mut new_effects);
                 }
-                Effect::GMsg(g_msg) => {
```

Comment:
I don't understand the broader system well enough to recognise what this change implies. A grep of `Effect::GMsg` came up with 4 hits:

1. This one
2. Pushing a message onto the global message queue [here](https://github.com/Ben-PH/seed/blob/30614b1d24356c2d2225e6f4cf526d9405ac3121/src/app/orders/container.rs#L102)
3. [10 lines up](https://github.com/Ben-PH/seed/blob/30614b1d24356c2d2225e6f4cf526d9405ac3121/src/app.rs#L247) pushing onto front as part of the `sink` function, which I understand we plan to remove.
4. does [nothing](https://github.com/seed-rs/seed/blob/master/src/app/effects.rs#L20)

So, in terms of utility, summarizing the use of global message

1. We are discarding it as an unused variable
2. We are storing the value into a queue.
3. As far as I can tell, this is the only place that the value is used. Everywhere else, it's private and seems to only come after calling `fn sink`
4. is effectively a no-op as far as GM is concerned

#### Interesting...
Aside from running `self.data.msg_listeners` callbacks, and running the `update` function, the code is the same, but not if we consider `OrdersContainer` construction because it always gives `ShouldRender::Render`. This means that all it does is call `self.schedule_render()` and return an empty queue. 

#### Possible issues?

Well, maybe. I can't see how the system uses the value that `::GMsg` wraps, and I feel a bit blind as to the motivations to put in `sink`in the first place. What are we throwing away here?

Reply:
`sink`, `Effect::GMsg`, `<...GMs..>`, `send_g_gms`, `perform_g_cmd`and `perform_g_cmd_with_handle` should be removed.

Global messages work the same like regular messages (`Msg` / `Ms`), but the whole app use only one type. This mechanism of inter-component communication has been replaced by `orders.subscribe / notify`. Everything  `GMsg`-related will be deleted.